README.md

```diff
@@ -28,5 +28,236 @@
 
 # Triton Inference Server
 
+[![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
 **NOTE: You are currently on the r23.12 branch which tracks stabilization
 towards the next release. This branch is not usable during stabilization.**
```

Comment:
Let's remove this line to do not come back to it on release date

Reply:
I would prefer keeping this line to avoid confusion among users. We can remove the line when 23.12 containers are released. We have seen users trying out frozen branches which are not yet released in the past. Its not a big change to be done post NGC release.